### PR TITLE
add pricing

### PR DESCRIPTION
**Pull request by thatmlopsguy**
More about https://paul.af/slash-pricing

Idea from @Pinjasaur

### CODE REVIEW

**Comment by Pinjasaur** [comment]
Love to see this project. Thanks for making it @rknightuk and thanks for the plug @thatmlopsguy. 🙂

File: pages/pricing.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: pricing
+description:  a page for those that want to buy your products can know how much they cost.
```
Comment (Pinjasaur) [inline pages/pricing.md:3]:
```suggestion
description: a page for those that want to buy your products so they can know the cost
```

nit: readability

File: pages/pricing.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: pricing
+description: a page for those that want to buy your products so they can know the cost
+creator_link: https://paul.af/pricing
```
Comment (Pinjasaur) [inline pages/pricing.md:4]: question: not sure if this should be this URL or https://paul.af/slash-pricing

Comment (rknightuk) [reply pages/pricing.md:4]: Yeah for creator links I try to link to the explainer page if I can

**Comment by rknightuk** [comment]
Thanks folks, should be on the site momentarily.

**Pull request merged**